ui/components/DashboardComponent.js

```diff
@@ -84,6 +84,9 @@ const styles = (theme) => ({
     height : "100%",
   },
 });
+async function fetchAllContexts(number) {
```

Comment:
If this number signifies `pageSize` then please rename the variable to `pageSize`.